src/test/ledger/env.ts

```diff
@@ -38,25 +43,25 @@ const PERUNART_NAME = "PerunArt";
 const PERUNART_SYMBOL = "PART";
 export const PERUNART_URI = "https://nifty.erdstall.dev/";
 const PERUN_FUNDS = utils.parseEther("100000").toBigInt();
-const gProvider = new MockProvider();
-const wallets = gProvider.getWallets();
 const OP = 0,
 	TEE = 1;
 
 export async function setupEnv(
 	numOfPrefundedAccounts: number = 1,
 	epochDuration: number = 3,
-	lprovider?: providers.Web3Provider,
-	lop?: Wallet,
+	ganacheOptions?: MockProviderOptions,
 	pacc?: Wallet,
```

Comment:
Does it even still make sense to pass `pacc?: Wallet`? Or does it cause inconsistencies?

This also now removes the ability to inject a `lprovider`? Do we use this in any other project?

Reply:
Taken from my commit message:

Also moved global variables into `setupEnv` function:
  * In hindsight unnecessary.
  * Passing a local provider can be helpful, but never was so far.
  * With the `lprovider` gone there is also no need for `lop` to be
    passed to the setup function.
    
Initially I did not remove `pacc` because it was independent from the other settings I deleted.

Thinking about it again, it **HAS** to be removed, because it effectively overrides the first `numOfPrefundedAccounts` option a user passes. So lets just purge this too.

Reply:
Sounds good. We better remove it and if we ever need this feature, we can add it, knowing how exactly.